src/_help/doc-beta.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Beta
+---
+
+- TOC
+{:toc}
+
+Use the `x-beta` property inside an Operation, a schema or a parameter object to identify it as beta.
+The `x-beta` property is a boolean.
+
+Add or remove this property is a strucutal change.
+A change on a beta component, can't be breaking.
+
+## Example usage
+
+The following screen capture shows the rendering of adding two code samples `cURL` and `Ruby` to your Operation.
+
+Hereunder is an example of a beta Operation, Schema and Parameter.
+
+```yaml
+  "paths": {
+    "/diffs": {
+      "post": {
+        "x-beta": true,
+        "description": "Create a diff between any two given API definitions",
+        "requestBody": {
+          "x-beta": true,
```

Comment:
Did we add the parsing of `x-beta` on Request body objects? I thought we only added it on `Operations` and `Schema`?

Reply:
Yes, my bad, you're right. I forgot to correct this.